screen/settings/about.js

```diff
@@ -92,27 +92,25 @@ const About = () => {
   };
 
   const handleOnTwitterPress = () => {
-    Linking.openURL('https://twitter.com/bluewalletio');
-  };
-
-  const handleOnDiscordPress = () => {
-    Linking.openURL('https://discord.gg/btWq2Aby2z');
+    Linking.openURL('https://twitter.com/DFX_Swiss');
   };
 
   const handleOnTelegramPress = () => {
-    Linking.openURL('https://t.me/bluewallethat');
+    Linking.openURL('https://t.me/DFXswiss_en');
   };
+
   const handleOnGithubPress = () => {
-    Linking.openURL('https://github.com/BlueWallet/BlueWallet');
+    Linking.openURL('https://github.com/DFXSwiss/btc-wallet');
   };
+
   const handleOnRatePress = () => {
     const options = {
-      AppleAppID: '1376878040',
-      GooglePackageName: 'io.bluewallet.bluewallet',
+      AppleAppID: '6443845399',
+      GooglePackageName: '',
```

Comment:
Hm, this is probably needed 🤔

Reply:
yes, it was empty on the other changes too. Thought that we would need to create an Android App first on Google PlayStore to have a correct package name to insert here.

Reply:
Hm, then it's probably just
![image](https://user-images.githubusercontent.com/85513542/213390772-0d4ecdbb-811f-4250-ac07-5494d3aab785.png)